### PR TITLE
RELATED: FET-56 Move Table from indigo-visualizations to react-components

### PR DESCRIPTION
**Pull request by vaclavbohac**
`5.1.0-ocetnik-doc-indigo-vis-table-in-ts-2018-05-03T09-10-18-738Z`

### CODE REVIEW

**Comment by gdgate** [comment]
Merge Failed.

This change or one of its cross-repo dependencies was unable to be automatically merged with the current state of its repository. Please rebase the change and upload a new patchset.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1480/ : SUCCESS in 24s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1218/ : FAILURE in 1m 34s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/691/ : FAILURE in 1m 44s

**Comment by vaclavbohac** [comment]
ok to test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1486/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1223/ : SUCCESS in 3m 04s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/696/ : SUCCESS in 2m 26s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1487/ : SUCCESS in 16s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1224/ : SUCCESS in 3m 00s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/697/ : SUCCESS in 2m 20s

**Comment by ocetnik** [comment]
extended test - storybook

**Comment by osklive-kacatko** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/504/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1489/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/431/ : UNSTABLE in 9m 33s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1491/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1225/ : SUCCESS in 7m 48s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/698/ : SUCCESS in 2m 51s

**Comment by ocetnik** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1494/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1227/ : SUCCESS in 1m 48s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/700/ : SUCCESS in 2m 41s

File: stories/visualizations/TableTransformation.tsx [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+        )
+    ))
+    .add('Totals edit mode', () => {
+        let tableRef: any;
+        const totals = generateTotals(['avg', 'nat', 'sum']);
+        const response = cloneDeep(EXECUTION_RESPONSE_2A_3M);
+        // set formatting of third metric - for null display "unknown"
+        const header: Execution.IHeader = response.executionResponse.dimensions[1].headers[0];
+        if (isMeasureGroupHeader(header)) {
+            header.measureGroupHeader.items[2].measureHeaderItem.format = '[=null][magenta]unknown';
```
Comment (petrjanu) [inline stories/visualizations/TableTransformation.tsx]: Tuto @ocetnik

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1497/ : SUCCESS in 46s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1230/ : SUCCESS in 6m 43s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/703/ : SUCCESS in 3m 39s

**Comment by vaclavbohac** [comment]
extended test - storybook

**Comment by osklive-kacatko** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/508/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1499/ : SUCCESS in 39s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/435/ : UNSTABLE in 11m 49s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1500/ : SUCCESS in 55s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1231/ : SUCCESS in 1m 26s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/704/ : SUCCESS in 2m 35s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1501/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1233/ : FAILURE in 27s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/706/ : FAILURE in 1m 30s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1514/ : SUCCESS in 31s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1242/ : FAILURE in 38s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/715/ : FAILURE in 1m 42s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/14/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1274/ : FAILURE in 2m 58s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/747/ : FAILURE in 1m 56s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/15/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1275/ : FAILURE in 5m 25s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/748/ : SUCCESS in 2m 33s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/17/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1277/ : FAILURE in 3m 04s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/750/ : SUCCESS in 3m 17s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/18/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1278/ : SUCCESS in 3m 57s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/751/ : SUCCESS in 6m 50s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/19/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1280/ : FAILURE in 9m 58s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/753/ : SUCCESS in 2m 52s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/22/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1284/ : SUCCESS in 2m 26s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/757/ : SUCCESS in 3m 54s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/23/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1285/ : SUCCESS in 3m 06s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/758/ : SUCCESS in 3m 39s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/24/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1286/ : SUCCESS in 3m 25s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/759/ : SUCCESS in 3m 15s

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/26/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/458/ : FAILURE in 12m 27s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/27/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1288/ : SUCCESS in 1m 34s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/761/ : SUCCESS in 2m 35s

**Comment by gdgate** [comment]
Merge Failed.

This change or one of its cross-repo dependencies was unable to be automatically merged with the current state of its repository. Please rebase the change and upload a new patchset.

**Comment by gdgate** [comment]
Merge Failed.

This change or one of its cross-repo dependencies was unable to be automatically merged with the current state of its repository. Please rebase the change and upload a new patchset.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/46/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1305/ : SUCCESS in 3m 24s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/778/ : SUCCESS in 2m 46s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/50/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1309/ : SUCCESS in 2m 49s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/782/ : SUCCESS in 3m 06s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/51/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1311/ : SUCCESS in 3m 13s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/784/ : SUCCESS in 3m 08s

**Comment by gdgate** [comment]
Merge Failed.

This change or one of its cross-repo dependencies was unable to be automatically merged with the current state of its repository. Please rebase the change and upload a new patchset.

**Comment by ocetnik** [comment]
extended test - storybook

**Comment by yenkins** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/533/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/54/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/461/ : UNSTABLE in 12m 55s

**Comment by ocetnik** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/55/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1314/ : SUCCESS in 1m 37s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/787/ : SUCCESS in 2m 56s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/534/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/56/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/462/ : SUCCESS in 11m 45s

**Comment by ocetnik** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/57/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1315/ : SUCCESS in 3m 12s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/788/ : SUCCESS in 2m 19s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/535/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/58/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/463/ : UNSTABLE in 12m 51s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/60/ : SUCCESS in 12s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1317/ : SUCCESS in 3m 33s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/790/ : SUCCESS in 2m 57s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/536/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/61/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/464/ : SUCCESS in 12m 37s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/62/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1318/ : SUCCESS in 1m 35s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/791/ : SUCCESS in 2m 34s

**Comment by gdgate** [comment]
Merge Failed.

This change or one of its cross-repo dependencies was unable to be automatically merged with the current state of its repository. Please rebase the change and upload a new patchset.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/63/ : SUCCESS in 20s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1320/ : SUCCESS in 3m 17s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/793/ : SUCCESS in 3m 06s

**Comment by ocetnik** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/64/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1321/ : FAILURE in 3m 58s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/794/ : SUCCESS in 3m 01s

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/65/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/465/ : FAILURE in 5m 30s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/66/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1322/ : FAILURE in 1m 34s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/795/ : SUCCESS in 3m 35s

**Comment by ocetnik** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/67/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1323/ : SUCCESS in 1m 45s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/796/ : SUCCESS in 3m 46s

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/68/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/466/ : FAILURE in 4m 50s

**Comment by ocetnik** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/69/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/467/ : FAILURE in 3m 13s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/70/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1324/ : SUCCESS in 3m 10s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/797/ : SUCCESS in 3m 01s

**Comment by ocetnik** [comment]
extended test - storybook

**Comment by yenkins** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/541/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/71/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/468/ : SUCCESS in 13m 55s

**Comment by ocetnik** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/72/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1325/ : SUCCESS in 3m 32s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/798/ : SUCCESS in 4m 04s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/542/artifact/test-storybook/html-report/index.html

**Pull request closed without merging**